### ISSUE TITLE
Nasty error message with non-existent SRA accession

### ISSUE THREAD

**Issue by jakelever** [boratyng/magicblast#3]
If the SRA accession doesn't exist, you get a strange error message. It'd be nice if it just told you that the accession doesn't exist.

```
(root) [CRCUSER9@login0a PSST]$ magicblast -sra NOTREAL -db mydb -out out.txt -outfmt tabular -parse_deflines T -num_threads 8
Error: NCBI C++ Exception:
    T0 "/home/coremake/release_build/build/PrepareRelease_Linux64-Centos_JSID_01_370586_130.14.18.6_9008__PrepareRelease_Linux64-Centos_1505331058/c++/compilers/unix/../../src/sra/readers/sra/vdbread.cpp", line 555: Error: ncbi::objects::CVDB::CVDB() - Cannot open VDB: NOTREAL (0x9be50398: directory not found while opening manager within virtual file system module)
```

**Comment by boratyng** [comment]
Thank you for the report. We are going to fix this soon.

**Comment by boratyng** [comment]
This is fixed in version 1.4.0.
